chef/cookbooks/provisioner/recipes/setup_base_images.rb

```diff
@@ -240,6 +240,15 @@
   package "tftp-server"
 when "suse"
   package "tftp"
+
+  # work around change in bnc#813226 which breaks
+  # read permissions for nobody and wwwrun user
+  directory tftproot do
+    recursive true
```

Comment:
https://docs.chef.io/resource_directory.html

its recursively creating the directory (aka first /srv, then /srv/tfpboot). Also the permissions are only set on the leaf directory, the rest stays untouched.